lib/irc/BridgedClient.js

```diff
@@ -327,6 +327,7 @@ BridgedClient.prototype.whois = function(nick) {
             resolve({
                 server: self.server,
                 nick: nick,
+                isOp: whois.operator !== undefined,
```

Comment:
I *think* this is set from `RPL_WHOISOPERATOR` which is the response command to `WHOIS`. This is whether they are an IRC operator rather than a chan op, which is what we really want to be checking here.